.github/workflows/base-windows.yml

```diff
@@ -52,10 +52,21 @@ on:
         type: string
         description: 'The issue number to report results to mandrel-integration-tests'
         default: "null"
+      build-stats-tag:
+        type: string
+        description: 'The tag to use for build stats upload of native tests (e.g. 22.3.0-dev-jdk17-mytest-patch-before)'
+        default: "null"
+      report-build-stats:
+        type: boolean
+        description: 'Whether or not to collect and report build stats (GraalVM 22.3.0+)'
+        default: true
```

Comment:
I believe this should be set to false by default. The nighlty workflow should set it to true as needed. When using a wrapper workflow the stats should get reported only if explicitly required by the user, not by default.

Actually, to make this work with the limited number of input options available for wrapper calls, we should instead report build stats if `build-stats-tag` is not null and not report them otherwise.